xml/System.IO.Compression/DeflateStream.xml

```diff
@@ -436,23 +447,32 @@ You use this constructor when you want to specify whether compression efficiency
             </Attribute>
           </Attributes>
         </Parameter>
-        <Parameter Name="state" Type="System.Object" Index="4" FrameworkAlternate="xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0" />
-      </Parameters>
       <Docs>
         <param name="buffer">The byte array to read the data into.</param>
         <param name="array">The byte array to read the data into.</param>
-        <param name="offset">The byte offset in <paramref name="array" /> at which to begin reading data from the stream.</param>
+        <param name="offset">The byte offset at which to begin reading data from the stream.</param>
```

Comment:
Since the parameter has a different name depending on the version, I decided just to remove it.